tasks/fine_tuning_sbert/src/sentence_transformer.py

```diff
@@ -199,42 +212,37 @@ def fit(self,
                 print(f'Epoch: {epoch}')
                 print(f"Best score: {self.best_score}")
                 print('=' * 60)
-                self.plot_train_val_accuracy(model_deets, output_path, training_acc_list)
                 return
 
             # removing the unnecessary first element in ACC_LIST that needed to be there for epoch 1
             if epoch == 0:
                 del self.acc_list[0]
 
-        if evaluator is None and output_path is not None:  # No evaluator, but output path: save final model version
+        # No evaluator, but output path: save final model version
+        if evaluator is None and output_path is not None:
```

Comment:
We still do this first before saving into wandb right? just making sure we're not double saving the model

Reply:
I realize that our ``evaluator`` parameter is never None so this clause is obsolete. Because the ``save`` function for the model **only** happens in the ``_eval_during_training`` method.
I will remove this clause.